tests/instancetype_test.go

```diff
@@ -946,102 +947,77 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			Expect(err).ToNot(HaveOccurred())
 			Expect(vmi.Spec.Domain.CPU.Cores).To(Equal(instancetype.Spec.CPU.Guest))
 		}
 
-		waitPVCBound := func(pvc *k8sv1.PersistentVolumeClaim) *k8sv1.PersistentVolumeClaim {
-			Eventually(func() bool {
-				var err error
-				pvc, err = virtClient.CoreV1().PersistentVolumeClaims(pvc.Namespace).Get(context.Background(), pvc.Name, metav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				return pvc.Status.Phase == k8sv1.ClaimBound
-			}, 180*time.Second, time.Second).Should(BeTrue())
-			return pvc
-		}
-
-		BeforeAll(func() {
+		BeforeEach(func() {
 			if !libstorage.HasCDI() {
 				Skip("instance type and preference inferFromVolume tests require CDI to be installed providing the DataVolume and DataSource CRDs")
 			}
 
-			blockStorageClass, blockStorageClassExists = libstorage.GetRWOBlockStorageClass()
-			if !blockStorageClassExists {
-				Skip("Skip test when RWOBlock storage class is not present")
-			}
-
 			By("Creating a VirtualMachineInstancetype")
 			instancetype = newVirtualMachineInstancetype(nil)
-			instancetype.Name = instancetypeName
-			instancetype, err = virtClient.VirtualMachineInstancetype(util.NamespaceTestDefault).
-				Create(context.Background(), instancetype, metav1.CreateOptions{})
+			instancetype, err = virtClient.VirtualMachineInstancetype(util.NamespaceTestDefault).Create(context.Background(), instancetype, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Creating a VirtualMachinePreference")
 			preference = newVirtualMachinePreference()
-			preference.Name = preferenceName
 			preferredCPUTopology := instancetypev1beta1.PreferCores
 			preference.Spec = instancetypev1beta1.VirtualMachinePreferenceSpec{
 				CPU: &instancetypev1beta1.CPUPreferences{
 					PreferredCPUTopology: &preferredCPUTopology,
 				},
 			}
-			preference, err = virtClient.VirtualMachinePreference(util.NamespaceTestDefault).
-				Create(context.Background(), preference, metav1.CreateOptions{})
+			preference, err = virtClient.VirtualMachinePreference(util.NamespaceTestDefault).Create(context.Background(), preference, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
-		})
-
-		BeforeEach(func() {
-			By("Creating an annotated source PVC")
-			sourcePVC = libstorage.NewPVC(pvcSourceName, "1Gi", blockStorageClass)
-			sourcePVCVolumeBlockMode := k8sv1.PersistentVolumeBlock
-			sourcePVC.Spec.VolumeMode = &sourcePVCVolumeBlockMode
-			sourcePVC.Labels = map[string]string{
+			By("Creating source DataVolume and PVC")
+			sourceDV = libdv.NewDataVolume(
+				libdv.WithNamespace(util.NamespaceTestDefault),
+				libdv.WithForceBindAnnotation(),
+				libdv.WithBlankImageSource(),
+				libdv.WithPVC(libdv.PVCWithAccessMode(k8sv1.ReadWriteOnce), libdv.PVCWithVolumeSize("1G")),
+			)
+			// TODO - Add withDefault{Instancetype,Preference}Label support to libdv
```

Comment:
Do this right away? Doesn's seem to be too big of a change.

Reply:
I'd like to backport this to `release-1.0` so I'd rather not include it here.